nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/tasks/ConnectableTask.java

```diff
@@ -251,8 +251,8 @@ public InvocationResult invoke() {
             } catch (final Throwable t) {
                 // Use ComponentLog to log the event so that a bulletin will be created for this processor
                 final ComponentLog procLog = new SimpleProcessLogger(connectable.getIdentifier(), connectable.getRunnableComponent());
-                procLog.error("{} failed to process session due to {}; Processor Administratively Yielded for {}",
-                    new Object[] {connectable.getRunnableComponent(), t, schedulingAgent.getAdministrativeYieldDuration()}, t);
+                procLog.error("Failed to process session due to {}; Processor Administratively Yielded for {}",
+                    new Object[] {t, schedulingAgent.getAdministrativeYieldDuration()}, t);
```

Comment:
Is this change required?

Reply:
Yes, this is an additional fix, I mentioned it in the description of the PR. The component name is automatically added by the logger, there's no need to add it here, it will occur twice.